connector/src/main/java/org/geysermc/connector/network/translators/inventory/translators/AnvilInventoryTranslator.java

```diff
@@ -75,4 +76,14 @@ public int javaSlotToBedrock(int slot) {
     public Inventory createInventory(String name, int windowId, WindowType windowType, PlayerInventory playerInventory) {
         return new AnvilContainer(name, windowId, this.size, windowType, playerInventory);
     }
+
+    @Override
+    public void updateProperty(GeyserSession session, Inventory inventory, int key, int value) {
+        if (key == 0) {
+            AnvilContainer anvilContainer = (AnvilContainer) inventory;
+            anvilContainer.setJavaLevelCost(value);
+            anvilContainer.setUseJavaLevelCost(true);
+            updateSlot(session, anvilContainer, 1);
+        }
```

Comment:
```suggestion
        // if key is not 0, return (idk what key being 0 means)
        if (key != 0) return;

        AnvilContainer anvilContainer = (AnvilContainer) inventory;
        anvilContainer.setJavaLevelCost(value);
        anvilContainer.setUseJavaLevelCost(true);
        updateSlot(session, anvilContainer, 1);
```

Just a simple cleanup, definitely needs a better comment!